qa/workunits/ceph-helpers.sh

```diff
@@ -1607,6 +1649,19 @@ function test_wait_background() {
     if [ ! -z "$pids" ]; then return 1; fi
 }
 
+function get_admin_socket()
+{
```

Comment:
all functions in ceph-helpers.sh are consistent with how the run_mon and run_osd setup the directories. If making get_admin_socket comply with that is bizarre in the context of test.sh, maybe the function should be defined at the beginning of test.sh instead (there are a few already in this case).